tests/mocks/JWT.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Google\Auth\Tests;
+
+class JWT
+{
+  public static function encode()
+  {
+    $args = func_get_args();
+    $method = 'JWT';
```

Comment:
well I'm not a professional developer but... oh crap, yes I am!!! Better fix that one before my boss sees it.